api/lib/domain/services/authentication/fwb-oidc-authentication-service.js

```diff
@@ -1,4 +1,4 @@
-import { v4 as uuidv4 } from 'uuid';
+import { randomUUID as uuidv4 } from 'crypto';
```

Comment:
Je sais que cela fait un peu + de modifications, mais je me demande si ça ne serait pas mieux d'y aller jusqu'au bout dans le switch de la lib vers le module Node standard et enlever le `as uuidv4`. Dans ce fichier et dans les autres.

Sachant que actuellement, en effet randomUUID génère un uuid v4, mais rien ne dit que dans une version autre de Node JS cela passe automatiquement par défaut à la v5 (rendant l'alias erroné sans faire exprès).

```suggestion
import { randomUUID } from 'crypto';
// Et du coup ligne 47
const uuid = randomUUID();
```

Reply:
Ca peut se faire dans une nouvelle PR

Reply:
Quel intérêt de découper ? Cette PR fait 16 lignes, et avec le changement proposé elle devrait pas dépasser les 30 lignes 😅

Reply:
Je suis en phase avec @Steph0 : je vais faire la modif :-)